.github/workflows/beam_PreCommit_Xlang_Generated_Transforms.yml

```diff
@@ -0,0 +1,176 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: PreCommit Xlang Generated Transforms
+
+on:
+  push:
+    tags: ['v*']
+    branches: ['master', 'release-*']
+    paths:
+      - 'model/**'
+      - 'sdks/python/**'
+      - 'sdks/java/expansion-service/**'
+      - 'sdks/java/core/**'
+      - 'sdks/java/io/**'
+      - 'sdks/java/extensions/sql/**'
+      - '!sdks/java/io/amazon-web-services/**'
```

Comment:
Is there a reason to explicitly exclude these directories? (This seems a long list that could easily get out of date, and the risk of some transform service depending on these in the future might be non-obvious.)

Reply:
Explicitly excluded them to avoid running on irrelevant paths. I figured we could remove paths from this list when introducing a new service to the config. 

However, yes this does end up being one more thing to keep track of when adding a new service

Reply:
It's a tradeoff between that and running tests unnecessarily. I would guess that these are not often edited directories, so maybe not worth the tradeoff.

Reply:
Removed